modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/wal/WalPathsTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache.persistence.db.wal;
+
+import org.apache.ignite.configuration.DataStorageConfiguration;
+import org.apache.ignite.configuration.IgniteConfiguration;
+import org.apache.ignite.configuration.PersistentStoreConfiguration;
+import org.apache.ignite.testframework.GridTestUtils;
+import org.apache.ignite.testframework.junits.common.GridCommonAbstractTest;
+
+import java.util.concurrent.Callable;
+
+/** */
+public class WalPathsTest extends GridCommonAbstractTest {
+    /** Test name. */
+    private final String testName = getClass().getSimpleName();
+
+    /** {@inheritDoc} */
+    @Override protected void afterTest() throws Exception {
+        super.afterTest();
+
+        stopAllGrids();
+    }
+
+    /**
+     * @return Ignite configuration with data storage.
+     * @throws Exception If failed.
+     */
+    private IgniteConfiguration getConfigurationWithDataStorage() throws Exception {
+        IgniteConfiguration cfg = super.getConfiguration();
+
+        DataStorageConfiguration dsCfg = new DataStorageConfiguration();
+
+        dsCfg.setWalPath(testName + "\\wal");
+        dsCfg.setWalArchivePath(testName + "\\wal");
+
+        cfg.setDataStorageConfiguration(dsCfg);
+
+        return cfg;
+    }
+
+    /**
+     * @return Ignite configuration with persistent store.
+     * @throws Exception If failed.
+     */
+    @Deprecated
+    private IgniteConfiguration getConfigurationWithPersistentStore() throws Exception {
+        IgniteConfiguration cfg = super.getConfiguration();
+
+        PersistentStoreConfiguration psCfg = new PersistentStoreConfiguration();
+
+        psCfg.setWalStorePath(testName + "\\wal");
+        psCfg.setWalArchivePath(testName + "\\wal");
+
+        cfg.setPersistentStoreConfiguration(psCfg);
+
+        return cfg;
+    }
+
+    /**
+     * @throws Exception If failed.
+     */
+    @Deprecated
+    public void testPersistentStoreWalAndArchivePaths() throws Exception {
```

Comment:
Remove this test. Yout shouldn't test deprecated configuration since you know that it will be converted to  DataStorageConfiguration.